services/QuillLMS/app/workers/copy_concept_results_to_responses_worker.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class CopyConceptResultsToResponsesWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: SidekiqQueue::MIGRATION
+
+  BATCH_SIZE=100000
```

Comment:
Small style nit, you can use the `_` symbol in Ruby integers to make large ones more readable, e.g.
```suggestion
  BATCH_SIZE = 100_000
```